packages/react-day-picker/src/components/Table/utils/getWeeks.test.ts

```diff
@@ -0,0 +1,33 @@
+import { enGB, enUS } from 'date-fns/locale';
+import { getWeeks } from './getWeeks';
+
```

Comment:
We need to "freeze the time" here as well (like in `Caption` with the tk package).

Reply:
I think we may only need to freeze time in the case that we are testing code that uses `new Date()`.

Reply:
Would effects happen even if the `new Date()` is used in the test? Anyway thanks for the heads up!